### PR TITLE
Fix various tests

### PR DESCRIPTION
**Pull request by npomaroli**
### General

* [ ] Added abstract that describes the change
* [ ] Added changelog entry to `/CHANGELOG.adoc`
* [ ] Ensured that the change is covered by tests
* [ ] Ensured that the change is documented in the docs

### On API Changes

* [ ] Checked if the changes are breaking or not
* [ ] Added GraphQL API if applicable
* [ ] Added Elasticsearch mapping if applicable

### CODE REVIEW

File: tests/tests-core/src/main/java/com/gentics/mesh/test/context/AbstractMeshTest.java [hunk 1]
```diff
@@ -101,7 +101,6 @@ public OkHttpClient httpClient() {
 
 	public String getJson(HibNode node) throws Exception {
 		InternalActionContext ac = mockActionContext("lang=en&version=draft");
-		ac.data().put(SharedKeys.PROJECT_CONTEXT_KEY, TestDataProvider.PROJECT_NAME);
```
Comment (plyhun) [inline tests/tests-core/src/main/java/com/gentics/mesh/test/context/AbstractMeshTest.java:104]: What is the purpose for this change?

Comment (npomaroli) [reply tests/tests-core/src/main/java/com/gentics/mesh/test/context/AbstractMeshTest.java:104]: The data should contain an instance of the project, not the name of the project stored with this key.
The call to mockActionContext() already puts the project into the data, but this is incorrectly overwritten here.

**Review by plyhun — APPROVED** [review]
